README.md

```diff
@@ -1,2 +1,9 @@
-# freshkite-picnic
-Freshkite picnic  project
+# Freshkite Picnic
+
+This repository contains the information related to the freshkite picnic
```

Comment:
This line is missing our mission for the picnic

Reply:
Fixed